docs/content/guides/operator/snapshots.mdx

```diff
@@ -60,8 +70,9 @@ Add an entry to the config file for `db-checkpoint-config`. Using Amazon's S3 se
 
 Add an entry to the config file for `db-checkpoint-config`. Using Amazon's S3 service as an example:
    ```yaml
-   state-snapshot-write-config:
-      object-store-config:
+   db-checkpoint-config:
+     state-snapshot-write-config:
+       object-store-config:
```

Comment:
When did we make this change?

Reply:
> Add an entry to the config file for `db-checkpoint-config`. Using Amazon's S3 service as an example:

I was just looking at the directive above this config block - should it not be nested?

Reply:
Checking your docs, I think this was just a bad copy paste that was made earlier - I'll fix

Reply:
Sounds good. Yeah `state-snapshot-write-config` should be a global config key in the node config: https://github.com/MystenLabs/sui/blob/e1ca530f346ebc13816472a65429448843ec5508/crates/sui-config/src/node.rs#L146